app/src/main/java/nl/tudelft/b_b_w/controller/TrustController.java

```diff
@@ -0,0 +1,94 @@
+package nl.tudelft.b_b_w.controller;
+
+import nl.tudelft.b_b_w.model.block.Block;
+
+/**
+ * Class to edit the trust value of a block
+ */
+class TrustController {
+
+    /**
+     * Regularization parameters
+     */
+    private static final int REGULARIZATION_TRANSACTION = 1;
+    private static final int REGULARIZATION_VERIFIED_IBAN = 3;
+
+    /**
+     * succesfulTransaction
+     * Calculates the new trust value given a succesful transaction
+     *
+     * @param block the given block
+     * @return the block with the new trust value
+     */
+    Block succesfulTransaction(Block block) {
+        final double newValue = distributionFunction(getX(block.getTrustValue()) +
+                REGULARIZATION_TRANSACTION);
+        block.setTrustValue(newValue);
+        return block;
+    }
+
+    /**
+     * failedTransaction
+     * Calculates the new trust value given a failed transaction
+     *
+     * @param block the given block
+     * @return the block with the new trust value
+     */
+    Block failedTransaction(Block block) {
+        final double newValue = checkCeiling(distributionFunction(getX(block.getTrustValue()) -
```

Comment:
since you only check whether the new value is under 0, does this mean that your max trust can be more than 100?